### ISSUE TITLE
Add Unit-tests to test the services

### ISSUE THREAD

**Issue by vibhcool** [loklak/loklak_server#1132]
- Issue type: Feature request

Loklak Server has Unit-Testing System. So there is a need to add unit tests to test the small services in the project.

**Comment by singhpratyush** [comment]
@vibhcool: I think that we've completed this part. Can this be closed now?

**Comment by vibhcool** [comment]
@singhpratyush , this is a need to add unit tests to the project. Only some of the classes are tested.

**Comment by singhpratyush** [comment]
But isn't this issue too broad? I think it is obvious that we need tests and we don't need to have an issue for that. It makes more sense if the issues for tests are specefic.

**Comment by AnshulMalik** [comment]
I agree @singhpratyush. We should split this issue into child issues or provide a checklist, so that people can know which services are to be tested.

**Comment by vibhcool** [comment]
@AnshulMalik , yes, you can mention this issue as parent issue like https://github.com/loklak/loklak_server/issues/1452
